server/resp/src/main/java/org/infinispan/server/resp/commands/json/JSONGET.java

```diff
@@ -0,0 +1,35 @@
+package org.infinispan.server.resp.commands.json;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.functional.FunctionalMap;
+import org.infinispan.functional.impl.FunctionalMapImpl;
+import org.infinispan.functional.impl.ReadOnlyMapImpl;
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespRequestHandler;
+import org.infinispan.server.resp.commands.Resp3Command;
+import org.infinispan.server.resp.serialization.Resp3Response;
+
+import io.netty.channel.ChannelHandlerContext;
+
+public class JSONGET extends RespCommand implements Resp3Command {
+
+   public JSONGET() {
+      super("JSON.GET", -2, 1, 1, 1);
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler, ChannelHandlerContext ctx,
+         List<byte[]> arguments) {
+      byte[] key = arguments.get(0);
+      FunctionalMap.ReadOnlyMap<byte[], Object> cache = ReadOnlyMapImpl
+            .create(FunctionalMapImpl.create(handler.typedCache(null)));
+            CompletionStage<byte[]> cs = cache.eval(key, view -> {
+               return (byte[])view.find().orElse(null);
```

Comment:
Is this not the same as just a GET?

Reply:
yeah, it just a base impl to write json.set unit tests. 
Full implementation will be in #13177

Reply:
mmm maybe we can add some code here and handle an ad hoc the exception: https://github.com/infinispan/infinispan/blob/1c83c4ef8f1506c1136d273264b5eb0a5a5898fe/server/resp/src/main/java/org/infinispan/server/resp/serialization/ResponseWriter.java#L159

Reply:
@rigazilla Wouldn't it be better to have the two PRs as a single, with a single commit? JSON.SET doesn't really have much use without JSON.GET and vice versa. And it would enable to write better better tests without having to fetch the whole string and parse it via a different parser?

Reply:
well maybe yes, also it avoids rebasing the get PR on top of this